src/utils/utils.ts

```diff
@@ -26,20 +27,44 @@ function makeCanApi(options: CanCommunityOptions) {
  * @param community_account
  * @param code_id
  */
```

Comment:
@danielAlvess remembers to update the document too.

Reply:
updated